server/controllers/export.js

```diff
@@ -60,6 +64,42 @@ const buildFilterQuery = (search) => {
   };
 };
 
+const populateAttribute = function({ components }) {
+  if (components) {
+    const populate = components.reduce((currentValue, current) => {
+      return { ...currentValue, [current.split(".").pop()]: { populate: "*" } };
+    }, {});
+    return { populate };
+  }
+  return { populate: "*" };
+}
+
+ const  getPopulateFromSchema = function (schema)  { 
+  let populate = Object.keys(schema.attributes).reduce((currentValue, current) => {
+    const attribute = schema.attributes[current];
+
+    if (!["dynamiczone", "component"].includes(attribute.type)) {
+      if(attribute.type === 'relation') {
+        return { ...currentValue, [current]: '*' };
+      }
+      return currentValue;
+    }
+    return {
+      ...currentValue,
+      [current]: populateAttribute(attribute),
+    };
+  }, {});
+
+  ['createdBy', 'updatedBy'].map( i => {
```

Comment:
`Object.defineProperty` complexifies the code. Use instead
```
i => '*'
```
Plus the value should be `{ populate: '*' }` instead of a simple `'*'`, which leads to the code:
```
i => ({ populate: '*' })
```